projects/valtimo/components/src/lib/components/timeline/timeline.component.html

```diff
@@ -16,7 +16,7 @@
 
 @if (items$ | async; as items) {
   <ul class="timeline mb-0 py-5">
-    @for (item of items; track item.customData.id) {
+    @for (item of items; track item.summaryTranslationKey) {
```

Comment:
summaryTranslationKey is not unique 
![image](https://github.com/user-attachments/assets/f5a5617a-c6ee-4b72-900e-6a608b0d72c7)